src/pudl/glue/ferc1_eia.py

```diff
@@ -76,406 +78,230 @@ def get_plant_map() -> pd.DataFrame:
-    against the already mapped plants. Any plants found in the database but not in the
-    list of mapped plants are returned.
+    """Pull all plants in the FERC Form 1 DBF and XBRL DB for given years.
 
     Args:
         pudl_settings: Dictionary containing various paths and database URLs used by
             PUDL.
-        years: Years for which plants should be compiled from the raw FERC Form 1 DB.
-
-    Returns:
-        A dataframe containing five columns: utility_id_ferc1, utility_name_ferc1,
-        plant_name, capacity_mw, and plant_table. Each row is a unique combination of
-        utility_id_ferc1 and plant_name, which appears in the FERC Form 1 DB, but not in
-        the list of manually mapped plants.
-    """
-    db_plants = get_db_plants_ferc1(pudl_settings, years).set_index(
-        ["utility_id_ferc1", "plant_name_ferc1"]
-    )
-    mapped_plants = get_mapped_plants_ferc1().set_index(
-        ["utility_id_ferc1", "plant_name_ferc1"]
-    )
-    new_plants_index = db_plants.index.difference(mapped_plants.index)
-    unmapped_plants = db_plants.loc[new_plants_index].reset_index()
-    return unmapped_plants
-
-
-def get_unmapped_utils_ferc1(ferc1_engine):
-    """Generate a list of as-of-yet unmapped utilities from the FERC Form 1 DB.
-
-    Find any utilities which do exist in the cloned FERC Form 1 DB,
-    but which do not show up in the already mapped FERC respondents.
-
-    Args:
-        ferc1_engine (sqlalchemy.engine.Engine): A database connection engine
-            for the cloned FERC Form 1 DB.
+        years: Years for which plants should be compiled.
 
     Returns:
-        pandas.DataFrame: with columns "utility_id_ferc1" and "utility_name_ferc1"
+        A dataframe containing columns ``utility_id_ferc1``, ``utility_name_ferc1``,
+        ``plant_name``, ``capacity_mw``, and ``plant_table``. Each row is a unique
+        combination of ``utility_id_ferc1`` and ``plant_name``.
     """
-    all_utils_ferc1 = (
-        pd.read_sql_table("f1_respondent_id", ferc1_engine)
-        .rename(
-            columns={
-                "respondent_id": "utility_id_ferc1",
-                "respondent_name": "utility_name_ferc1",
-            }
+    # Validate the input years:
+    _ = pudl.settings.Ferc1Settings(years=list(years))
+
+    plant_tables = [
+        # "plants_hydro_ferc1",
+        # "plants_pumped_storage_ferc1",
+        # "plants_small_ferc1",
+        "plants_steam_ferc1",
+        "fuel_ferc1",
+    ]
+    ferc1_settings = pudl.settings.Ferc1Settings(tables=plant_tables)
+    # Extract FERC form 1
+    ferc1_dbf_raw_dfs = pudl.extract.ferc1.extract_dbf(
+        ferc1_settings=ferc1_settings, pudl_settings=pudl_settings
+    )
+    # Extract FERC form 1 XBRL data
+    ferc1_xbrl_raw_dfs = pudl.extract.ferc1.extract_xbrl(
+        ferc1_settings=ferc1_settings, pudl_settings=pudl_settings
+    )
+    plant_dfs: list[pd.DataFrame] = []
+    for table in plant_tables:
```

Comment:
And you didn't end up having to much with the special case of the `fuel_ferc1` dependency since that only comes into the (standard) `transform_main()` method, right?